video_emotion/emotionTagger/t/test_face_emotion_extraction.py

```diff
@@ -0,0 +1,77 @@
+from video_emotion.emotionTagger.face_emotion_extraction import classify_face
+from video_emotion.emotionTagger.face_emotion_extraction import get_labels
+from video_emotion.emotionTagger.face_emotion_extraction import preprocess_input
+import numpy as np
+import cv2
+import os
+
+def test_angry_face():
+    dirname = os.path.abspath(os.path.dirname(__file__))
+    filename = os.path.join(
+        dirname,
+        "test_face_emotion_extraction_data/angry1.png")
+    im = cv2.imread(filename)
+    faceData = classify_face(im)
+    assert len(faceData) == 7
+    assert faceData[0] > 0.5
+
+def test_happy_1():
+    dirname = os.path.abspath(os.path.dirname(__file__))
+    filename = os.path.join(
+        dirname,
+        "test_face_emotion_extraction_data/happy1.png")
```

Comment:
Like this instead?
```python
filename = os.path.join(
                        dirname,
                        "test_face_emotion_extraction_data/angry1.png")
```

Reply:
Still don't agree that Casper's edit is nicer.